.changeset/sharp-kiwis-own.md

```diff
@@ -0,0 +1,6 @@
+---
+'@rsbuild/plugin-module-federation': patch
```

Comment:
https://github.com/web-infra-dev/rslib/blob/main/.changeset/config.json#L8

We should modify `config.json` to replace `rsbuild-plugin-dts` with `rsbuild-*`, thus enabling to bump fixed version for `@rsbuild/plugin-module-federation`.